scripts/lib/sphinx/addFrontMatter.ts

```diff
@@ -20,7 +20,14 @@ export function addFrontMatter<T extends SphinxToMdResult>(
 ): void {
   for (let result of results) {
     let markdown = result.markdown;
-    if (result.meta.python_api_name) {
+    if (result.meta.hardcoded_frontmatter) {
```

Comment:
The order of this `if` matters. If you hardcode the frontmatter, we use that no matter what. That will be important when we fix https://github.com/Qiskit/documentation/issues/66 to add a fallback like using the filename. We want to make sure we still use the hardcoded metadata for these two index files.